README.md

```diff
@@ -229,6 +229,22 @@ peerConnection.addEventListener('track', event => {
 signalingClient.open();
 ```
 
+##### Add adapter.js to increase compatibility between browsers
+
+In your html, add the following:
+
+```
+<head>
+  ...
+  <script src="https://webrtc.github.io/adapter/adapter-latest.js"></script>
```

Comment:
adaptor.js can be added in a variety of ways and a script tag is not the most common, so I think that it would be better to just direct customers to the adaptor.js github page for installation instructions.

Reply:
Makes sense.